src/constants/plugins/buySellPlugins.js

```diff
@@ -65,6 +65,14 @@ export const allPlugins: Array<BuySellPlugin> = [
     supportEmail: 'support@sendwyre.com',
     permissions: ['camera']
   },
+  {
+    pluginId: 'co.edgesecure.bity',
```

Comment:
This pluginId sucks.

Reply:
co.edgesecure.bity mirrors what we did with wyre and simplex. I am open to suggestions.